README.md

```diff
@@ -1,5 +1,3 @@
-# W3C WebDriver Specification
```

Comment:
I would like the top level one, the build/validate ones can go since they
don't offer much.

On Thu, Feb 19, 2015 at 9:27 AM, Andreas Tolfsen <notifications@github.com>
wrote:

> In README.md
> <https://github.com/w3c/webdriver/pull/12#discussion_r24974403>:
>
> > @@ -1,5 +1,3 @@
> > -# W3C WebDriver Specification
>
> It looked quite odd with just a sentence under each heading, but I can add
> them all back in if you'd like.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/webdriver/pull/12/files#r24974403>.
>